pkg/services/live/pipeline/data_output_loki.go

```diff
@@ -0,0 +1,38 @@
+package pipeline
+
+import (
+	"context"
+	"time"
+)
+
+// LokiDataOutput passes processing control to the rule defined
+// for a configured channel.
+type LokiDataOutput struct {
+	lokiWriter *lokiWriter
+}
+
+func NewLokiDataOutput(endpoint, user, password string) *LokiDataOutput {
+	return &LokiDataOutput{
+		lokiWriter: newLokiWriter(endpoint, user, password),
+	}
+}
+
+const DataOutputTypeLoki = "loki"
+
+func (out *LokiDataOutput) Type() string {
+	return DataOutputTypeLoki
+}
+
+func (out *LokiDataOutput) OutputData(_ context.Context, vars Vars, data []byte) ([]*ChannelData, error) {
+	if out.lokiWriter.endpoint == "" {
+		logger.Debug("Skip sending to Loki: no url")
+		return nil, nil
+	}
+	err := out.lokiWriter.write(LokiStream{
+		Stream: map[string]string{"channel": vars.Channel},
```

Comment:
@ryantxu here is where I am not sure what we should put to identify our log entries.

Reply:
In theory it's possible to define some static labels or dynamic labels in output settings.

Reply:
this is great for now -- soon we should have both static+dynamic labels that could apply here and to loki